documentation/release_notes.markdown

```diff
@@ -16,6 +16,20 @@ Released 22 July 2019
 
 ### New features
 
+- This release includes an upgrade to the latest release of Jetty's 9.4 series. With this update, you may see "weak cipher" warnings about ciphers that were previously enabled by default. Puppet Server now defaults to stronger FIPS-compliant ciphers, but you must first remove the weak ciphers.
+
+The ciphers previously enabled by default have not been changed, but are considered weak by the updated standards. Remove the weak ciphers by removing the `cipher-suite` configuration section from the `webserver.conf`. After you do this, Puppet Server uses the FIPS-compliant ciphers instead. This release includes the weak ciphers for backward compatibility only.
```

Comment:
I think this and the following paragraphs need to be indented to be included in the bullet point format.